satellite/rewards/display.go

```diff
@@ -78,9 +80,18 @@ func (offers Offers) OrganizeOffersByType() OfferSet {
 func createPartnerSet() PartnerSet {
 	partners := LoadPartnerInfos()
 	var ps PartnerSet
-	for _, partner := range partners {
+	keys := make([]string, len(partners))
+	i := 0
+	for k := range partners {
```

Comment:
It looks like you could have written the more typical `for i := 0; k := range partners; i++ {`